int8 id
int8 object_size
int8 human_action
int8 human_expression